### ISSUE TITLE
Open image app when opening via writeily

### ISSUE THREAD

**Issue by kud** [plafue/writeily-pro#156]
I stock some images in my writeily folder and sometimes I explore them via writeily but it crashes the app. It should at least start the image app.

**Comment by homlett** [comment]
I totally agree with that. I've the same problem with .pdf files. When Writeily don't know to read a file, it should open the android dialog to open with an another app.

That would be very convenient.
